### PR TITLE
CHE-13358:Update dashboard dev image.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What does this PR do?
https://github.com/eclipse/che/issues/13358

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Review by evidolob — APPROVED** [review]

File: dockerfiles/dashboard-dev/Dockerfile [hunk 1]
```diff
@@ -8,14 +8,14 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM node:8.15.0
+FROM node:8.15.1
```
Comment (benoitf) [inline dockerfiles/dashboard-dev/Dockerfile]: maybe we should use 8.16.0 which has a newest yarn version ?

Comment (AndrienkoAleksandr) [reply dockerfiles/dashboard-dev/Dockerfile]: Done. Tested with dashboard and workspace loader. Workspace loader uses this image in the dev-file too.

**Review by benoitf — APPROVED** [review]

File: dockerfiles/dashboard-dev/Dockerfile [hunk 2]
```diff
@@ -8,14 +8,14 @@
...
+FROM node:8.16.0
 
 RUN apt-get update && \
     apt-get install -y git \
     && apt-get -y clean \
     && rm -rf /var/lib/apt/lists/* \
-    && echo fs.inotify.max_user_watches=524288 | tee -a /etc/sysctl.conf
-
+    && echo fs.inotify.max_user_watches=524288 | tee -a /etc/sysctl.conf \
+    && yarn global upgrade yarn@1.13.0
```
Comment (benoitf) [inline dockerfiles/dashboard-dev/Dockerfile]: do we need this yarn upgrade ?

Comment (AndrienkoAleksandr) [reply dockerfiles/dashboard-dev/Dockerfile]: I think we don't need.

**Review by ashumilova — APPROVED** [review]

**Pull request merged**